servicebus/data-plane/azure-servicebus/src/main/java/com/microsoft/azure/servicebus/primitives/RequestResponseLink.java

```diff
@@ -626,15 +627,19 @@ public void onReceiveComplete(Delivery delivery) {
 
             // Return response in a separate thread so reactor thread is free to handle reactor events
             final Message finalResponseMessage = responseMessage;
-            MessagingFactory.INTERNAL_THREAD_POOL.submit(() -> {
-                String requestMessageId = (String)finalResponseMessage.getCorrelationId();
-                if (requestMessageId != null) {
-                    TRACE_LOGGER.debug("RequestRespnseLink received response for request with id :{}", requestMessageId);
-                    this.parent.completeRequestWithResponse(requestMessageId, finalResponseMessage);
-                } else {
-                    TRACE_LOGGER.warn("RequestRespnseLink received a message with null correlationId");
-                }
-            });
+            try {
```

Comment:
[SpotBugs-P3]
Exceptional return value of java.util.concurrent.ExecutorService.submit(Runnable) ignored in com.microsoft.azure.servicebus.primitives.RequestResponseLink$InternalReceiver.onReceiveComplete(Delivery)

Add try and catch block will handling the exception

Reply:
Should double check with service team if this is what they want to do with those exceptions.

Reply:
No longer an issue. Will suppress it.